### PR TITLE
Add tag command

### PR DESCRIPTION
**Pull request by NyCodeGHG**

### CODE REVIEW

**Comment by NyCodeGHG** [comment]
Tag Commands:

- [x] create-tag
- [x] create-alias
- [x] edit-tag
- [ ] info
- [x] delete-tag
- [ ] delete-alias
- [x] list
- [x] from
- [x] search
- [ ] raw
- [ ] transfer

**Comment by DRSchlaubi** [comment]
for the raw command just post the tag to hastebin

File: src/main/kotlin/de/nycode/bankobot/commands/tag/commands/TransferTagCommand.kt [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+                    )
+                }
+                Emojis.whiteCheckMark.unicode -> {
+                    deleteAllReactions()
+                    val newTag = tag.copy(author = member.id)
+                    BankoBot.repositories.tag.save(newTag)
+                    editEmbed(
+                        Embeds.success(
+                            "Transfer erfolgreich!",
+                            "Der Tag \"${tag.name}\" wurde erfolgreich von ${message.author?.mention} zu ${member.mention} transferiert!"
```
Comment (warriorzz) [inline src/main/kotlin/de/nycode/bankobot/commands/tag/commands/TransferTagCommand.kt:108]: Checks failed, :/

File: src/main/kotlin/de/nycode/bankobot/commands/tag/commands/CreateAliasCommand.kt [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+import org.litote.kmongo.eq
+
+@PublishedApi
+@AutoWired
+@ModuleName(TagModule)
+internal fun createAliasCommand(): CommandSet = command("create-alias") {
+    description("Alias erstellen.")
+
+    invoke(WordArgument.named("tag"), WordArgument.named("alias")) { tagName, aliasName ->
+        val tag = findTag(tagName) ?: return@invoke
```
Comment (DRSchlaubi) [inline src/main/kotlin/de/nycode/bankobot/commands/tag/commands/CreateAliasCommand.kt:59]: you should make a tag argument. You can just use Argument.map and filter for it see docs and jdoodle command

**Review by DRSchlaubi — APPROVED** [review]

**Review by DRSchlaubi — CHANGES_REQUESTED** [review]
I literally can't even click request changes

**Comment by NyCodeGHG** [comment]
Please do not merge yet

**Review by warriorzz — APPROVED** [review]

**Pull request merged**